src/content/datachannel/datatransfer/js/main.js

```diff
@@ -168,6 +175,14 @@ function onReceiveMessageCallback(event) {
   receiveProgress.value = receivedSize;
 
   if (receivedSize === bytesToSend) {
+    var bitrate = Math.round(receivedSize * 8 /
+        ((new Date()).getTime() - timestampStart));
```

Comment:
You might want to use `window.performance.now()` for this (more information [here](http://www.html5rocks.com/en/tutorials/webperformance/usertiming/)).